content/departments/cloud/trial_mi.md

```diff
@@ -76,6 +76,33 @@ When customer has decided to sign the deal, instance requestor will create [Mana
 
 When trial expires and customer do not wish to sign the deal, instance requestor will open [Teardown Managed Instance request](./index.md#managed-instance-requests)
 
+## Trial Managed Instance creation flow
+
+1. New [trial managed instance request issue](https://github.com/sourcegraph/customer/issues/new?assignees=&labels=team%2Fcloud%2C+mi%2Cmi%2Fnew-instance-request%2C+cloud-trial&template=new_trial_managed_instance.md&title=New+Trial+Managed+Instance+request%3A+%5Bdomain+name%5D) is open by CE or growth team (DRI Malo Marrec)
```

Comment:
[Minior] I think we have a standard of starting team names with Capital letters also unless I'm wrong the team name is `Product Growth`. If the team already has a handbook page I would link it here.

Reply:
you can also link Malo to either slack handle or handbook team page